executor/window.go

```diff
@@ -337,7 +337,15 @@ func (p *rowFrameWindowProcessor) appendResult2Chunk(ctx sessionctx.Context, row
 			if slidingWindowAggFunc != nil && initializedSlidingWindow {
 				err = slidingWindowAggFunc.Slide(ctx, rows, lastStart, lastEnd, shiftStart, shiftEnd, p.partialResults[i])
 			} else {
-				_, err = windowFunc.UpdatePartialResult(ctx, rows[start:end], p.partialResults[i])
+				// For MinMaxSlidingWindowAggFuncs, it needs the absolute value of each start of window, to compare
+				// whether elements inside deque are out of current window.
+				if minMaxSlidingWindowAggFunc, ok := windowFunc.(aggfuncs.MaxMinSlidingWindowAggFunc); ok {
+					// Store start inside MaxMinSlidingWindowAggFunc.windowInfo
+					minMaxSlidingWindowAggFunc.SetWindowStart(start)
+					_, err = minMaxSlidingWindowAggFunc.UpdatePartialResult(ctx, rows[start:end], p.partialResults[i])
+				} else {
+					_, err = windowFunc.UpdatePartialResult(ctx, rows[start:end], p.partialResults[i])
```

Comment:
Original `windowFunc.UpdatePartialResult(.., rowsInGroup []chunk.Row, ...)` only accepts array of rows, but does not pass in the absolute index. 

Example:

```
start, end := 127, 129
_, err = windowFunc.UpdatePartialResult(ctx, rows[127:129], p.partialResults[i])
```

Inside function `UpdatePartialResult`, we only accepts an array of length 2, but have no idea about the start of current window(127).

To fix such issue, we have two methods, either change the function signature directly(complicates code) or pass necessary info indirectly(though adding new struct to `MaxMinSlidingWindowAggFunc`). I implemented the latter one.